elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -411,6 +429,44 @@ class PfeSearchDroplist extends PFElement {
     return "pfe-search-droplist.scss";
   }
 
+  static get properties() {
```

Comment:
Can we go ahead and pull out PfeSearchDroplist into its own file?  The compiler is updated to handle this now.

Reply:
Yeah, in a different PR though.

Reply:
New issue created